http-core/src/main/scala/org/apache/pekko/http/impl/engine/http2/Http2StreamHandling.scala

```diff
@@ -445,6 +445,11 @@ private[http2] trait Http2StreamHandling extends GraphStageLogic with LogHelper
       case _: WindowUpdateFrame =>
         // We're not planning on sending any data on this stream anymore, so we don't care about window updates.
         this
+      case rst: RstStreamFrame =>
+        val headers = ParsedHeadersFrame(rst.streamId, endStream = false, Seq((":status", "429")), None)
+        dispatchSubstream(headers, Right(Source.failed(new PeerClosedStreamException(rst.streamId, rst.errorCode))),
+          correlationAttributes)
+        Closed
```

Comment:
should I handle `rst: RstStreamFrame` is other states? should I change other places that handle `RstStreamFrame` to return the fake response?